test/WorkflowCore.UnitTests/Services/DefinitionStorage/DefinitionLoaderTests.cs

```diff
@@ -43,6 +43,16 @@ public void ParseDefinition()
             A.CallTo(() => _registry.RegisterWorkflow(A<WorkflowDefinition>.That.Matches(MatchTestDefinition, ""))).MustHaveHappened();
         }
 
+        [Fact(DisplayName = "Should parse definition")]
+        public void ParseDefinitionPropertyDynamic()
```

Comment:
Should we also test that the nested output properties are getting mapped according to your change?

Reply:
Ok , I add nested output properties in  test